### DISCUSSION TITLE
List entries that use an Asset (feature request)

### DISCUSSION THREAD

**Discussion by proimage** [craftcms/cms#8552] [Ideas]
It would be great to be able to see which entries use an asset, perhaps as an extra "Alert" screen before a deletion or replacement action, or perhaps as a separate menu item altogether.

**Comment by jsunsawyer** [comment]
@proimage I just did a Craft 2 > 3 content migration and cleaned up unused assets using the [Asset Usage](https://github.com/born05/craft-assetusage/blob/master/README.md) plugin.

Here's what you get:
![screen shot 2018-06-05 at 1 59 16 pm](https://user-images.githubusercontent.com/1007191/41002470-b6fe9166-68c8-11e8-8165-51b24b462252.png)

Without something like this, I'm scared to remove anything. It's doesn't account for images used in redactor fields (and probably some other places), but it's really helpful.

Having this + a full reference of instances were assets are used  would be super helpful to ensure you don't make unintended changes.
